scripts/zones/Sea_Serpent_Grotto/DefaultActions.lua

```diff
@@ -2,6 +2,7 @@ local ID = zones[xi.zone.SEA_SERPENT_GROTTO]
 
 return {
     ['qm1'] = { messageSpecial = ID.text.NOTHING_OUT_OF_ORDINARY },
+    ['qm5'] = { messageSpecial = ID.text.NOTHING_OUT_OF_ORDINARY },
```

Comment:
Can you please align all of those ='s in each line here? Not sure how this got out of control